tests/unit/type/stream.tcl

```diff
@@ -914,15 +916,17 @@ start_server {tags {"stream needs:debug"} overrides {appendonly yes aof-use-rdb-
     }
 
     test {Stream can be rewrite into AOF correctly after XDEL lastid} {
-        r XSETID mystream 0-0
+        r DEL mystream
+        r XADD mystream 0-1 a b
+        r XSETID mystream 0-1
```

Comment:
i don't understand why you had to modify this test (and the one above).
were they unstable due to using `*` in previous tests?
or was it because you wanted to run them with `--only`?